PatrolRewardService/PatrolRewardService/ContextService.cs

```diff
@@ -376,8 +394,12 @@ public async Task<List<TxId>> StageTransactions(NineChroniclesClient client, int
         return result;
     }
 
-    public async Task<int> InvalidTxCount()
+    /// <summary>
+    /// Retrieves the count of pending transactions in the database.
+    /// </summary>
+    /// <returns>The number of pending transactions.</returns>
+    public async Task<int> PendingTxCount()
     {
-        return await _dbContext.Transactions.CountAsync(t => t.Result == TransactionStatus.INVALID);
+        return await _dbContext.Transactions.CountAsync(t => t.Result == TransactionStatus.INVALID || t.Result == TransactionStatus.STAGING);
```

Comment:
Tx may fail (lack of token balance, etc.). Don't need to catch failed Tx.s?

Reply:
Since this query is related to missing(or stucked) tx. maybe appropriate to count the failed tx separately.